flink-end-to-end-tests/flink-end-to-end-tests-common/src/main/java/org/apache/flink/tests/util/AutoClosableProcess.java

```diff
@@ -115,6 +115,22 @@ public void runBlocking(final Duration timeout) throws IOException {
 			}
 		}
 
+		public void runBlockingWithRetry(final int maxRetries, final Duration timeoutOfTry) throws IOException {
+			int retries = 0;
+			while (true) {
+				try {
+					runBlocking(timeoutOfTry);
+					break;
+				} catch (Throwable e) {
+					if (++retries > maxRetries) {
+						LOG.info("Exhausted to run the blocking process: {}. maxRetries={}, timeoutOfTry={} (ms)",
```

Comment:
should be included in the exception message like in `runBlocking`

Reply:
OK